pkgs/applications/networking/mailreaders/hasmail/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkgconfig
+, gobject-introspection
+, pango
+, cairo
+, gtk2
+}:
+
+buildGoModule rec {
+  pname = "hasmail";
+  version = "2019-08-24";
+  name = "${pname}-${version}-unstable";
+
+  src = fetchFromGitHub {
+    owner = "jonhoo";
+    repo = "hasmail";
+    rev = "eb52536d26815383bfe5990cd5ace8bb9d036c8d";
+    sha256 = "1p6kwa5xk1mb1fkkxz1b5rcyp5kb4zc8nfif1gk6fab6wbdj9ia1";
+  };
+
+  modSha256 = "0z3asz7v1izg81f9xifx9s2sp5hly173hajsn9idi3bkv0h78is2";
+
+  buildInputs = [
+    pkgconfig
+    gobject-introspection
```

Comment:
Do you need `wrapGAppsHook`? I've never looked at this program so I can't say. See https://github.com/NixOS/nixpkgs/pull/43150/files for documentation if required.

Reply:
No, you were right. I wonder why I thought I'd need it in the first place..